Bottom: b37708bc3850c579c81740824826af81df691a68
Top:    62b76d7726a2acec514f40237e82354c2d36de47
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-03-06 14:59:08 +0000

DEBUG: Polygon parentage on


---

diff --git a/src/polygon1.c b/src/polygon1.c
index 8041c05..efece45 100644
--- a/src/polygon1.c
+++ b/src/polygon1.c
@@ -2964,13 +2964,14 @@ poly_Boolean_free (POLYAREA * ai, POLYAREA * bi, POLYAREA ** res, int action)
 
   *res = NULL;
 
-#if 0
+#if 1
   /* Make copies for tracking polygon parentage (DEBUG) */
   if (!poly_M_Copy0 (&a_copy, a) || !poly_M_Copy0 (&b_copy, b))
       return err_no_memory;
-#endif
+#else
   a_copy = NULL;
   b_copy = NULL;
+#endif
 
   /* Move the parentage information over onto the copy */
   if (a_copy != NULL)
